app/presenters/tree_builder_configuration_manager.rb

```diff
@@ -1,25 +1,45 @@
-class TreeBuilderForeman < TreeBuilder
+class TreeBuilderConfigurationManager < TreeBuilder
   private
 
   def tree_init_options(_tree_name)
-    {:leaf => "ManageIQ::Providers::Foreman::ConfigurationManager"}
+    {:full_ids => true,
+     :open_all => true,
+     :leaf     => "ManageIQ::Providers::ConfigurationManager"}
   end
 
   def set_locals_for_render
     locals = super
-    locals.merge!(:id_prefix => 'pt_')
+    locals.merge!(:id_prefix  => 'pt_', :autoload   => true)
   end
 
   # Get root nodes count/array for explorer tree
   def x_get_tree_roots(count_only, _options)
-    objects = rbac_filtered_objects(ManageIQ::Providers::Foreman::ConfigurationManager.order("lower(name)"), :match_via_descendants => %w(ConfiguredSystem))
-    count_only_or_objects(count_only, objects)
+    objects = []
+    objects.push(:id      => "fr",
+                 :tree    => "fr_tree",
+                 :text    => "Foreman Providers",
+                 :image   => "folder",
+                 :tip     => "Foreman Providers",
+                 :expand  => true)
+    objects.push(:id      => "at",
+                 :tree    => "at_tree",
+                 :text    => "Ansible Tower Providers",
+                 :image   => "folder",
+                 :tip     => "Ansible Tower Providers",
+                 :expand  => true)
+    count_only_or_objects(count_only, objects, nil)
+  end
+
+  def x_get_tree_cmat_kids(object, count_only)
```

Comment:
It is unused in this initial step. Will be used when the equivalent for the configuration profiles are added.